web/openapi/openapi.yaml

```diff
@@ -411,21 +449,25 @@ paths:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/LaunchParameters"
+              $ref: '#/components/schemas/LaunchParameters'
       responses:
         200:
           description: Success.
           content:
             application/json:
               schema:
-                type: object
-                properties:
-                  id:
-                    type: string
-                    description: (If the pipeline does not end with a predefined sink) The ID of the successfully created pipeline.
-                  deployable:
-                    type: boolean
-                    description: (If the pipeline ends with a predefined sink) Returns this property with the value `true`.
+                oneOf:
+                  - $ref: '#/components/schemas/Launch Response For Pipeline With Sink'
+                  - $ref: '#/components/schemas/Launch Response For Pipeline Without Sink'
```

Comment:
That is because the schema names are displayed in the API documentation examples:
![image](https://github.com/tenzir/tenzir/assets/29815238/4fc19fc1-acd9-4964-9c85-688ebc04e8ce)